src/locales/zh.json

```diff
@@ -422,6 +422,10 @@
                 "vendor": "供应商"
             }
         },
+        "preview_room": {
```

Comment:
Please do not put empty locales because it won't fall back to the English version if you do that, it will just display an empty string.

Reply:
Oh, this is a copypasta of blank values from lines 524-526 (see below in the split diff view), I thought these were left blank intentionally, even though it doesn't make much sense now that I look at it.

Reply:
It looks like there are a lot of blank values in `src/locales/zh.json`, were they left there intentionally, e.g. to make translation progress easier?